### DISCUSSION TITLE
Unable to update custom fields in form

### DISCUSSION THREAD

**Discussion by gabrielevaudano** [Budibase/budibase#6099] [Help]
**Problem**: When I try to update many tables from one form using a custom schema and many repeaters with relationships, it breaks and tells me "Bad Gateway".

**My idea**: Maybe I should insert in "Save row" every id (already generated when the row has been created) before saving other fields I am trying to update?

**Question**: There is a solution to save every field in each correct table starting from only one form with a custom schema, where I set with "DefaultValue" each field in every repetitor?

Thank you for every answer. Your help is really appreciated.

https://user-images.githubusercontent.com/61832007/170817819-e0fdb001-864b-4819-a1e9-d432c8b43ac4.mp4

**Comment by melohagan** [comment]
Hi Gabriele,

Just to clarify, when you say you're getting a "Bad Gateway", you mean "Bad Request" as shown in the video?

If you open the DevTools, does it show any error messages on the request? Otherwise you can open the browser network tab (Fn+F12 on chrome).
It may simply be that you're trying to save a form that has a required field which you could add form validation for.
